internal/explorer.point/src/components/LinkPointToSol.jsx

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from 'react';
+import Swal from 'sweetalert2';
+import { useAppContext } from '../context/AppContext';
+
+const USER_REJECTION_CODE = 4001;
+const LS_KEY = 'point_to_sol_msg';
+const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+const styles = {
+    container: {
+        borderRadius: 4,
+        boxShadow: '0px 7px 15px 0 rgba(0, 0, 0, .15)',
+        padding: 16,
+        marginTop: 24,
+        color: 'rgb(124, 124, 124)',
+        display: 'grid',
+        gridTemplateColumns: '1fr 200px',
+        alignItems: 'center',
+        gridColumnGap: 42,
+    },
+    description: {
+        margin: 0,
+    },
+};
+
+const LinkPointToSol = () => {
+    const { walletIdentity } = useAppContext();
+    const [hasPointAddress, setHasPointAddress] = useState(true);
+    const [display, setDisplay] = useState(
+        () => localStorage.getItem(LS_KEY) !== 'hide',
+    );
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const { data } = await window.point.identity.identityToOwner({
+                    identity: walletIdentity,
+                });
+                if (!data.pointAddress || data.pointAddress === EMPTY_ADDRESS) {
+                    setHasPointAddress(false);
+                }
+            } catch (err) {
+                console.error(err);
+            }
+        }
+
+        if (
+            walletIdentity.endsWith('.sol') &&
+            window.point.point.link_point_to_sol
```

Comment:
This is here to make sure that the version of Point SDK in use exposes this method. This way deployments of Point Engine and Point SDK are not coupled.